staging/src/k8s.io/component-base/logs/json/json.go

```diff
@@ -57,6 +100,8 @@ func (l *zapLogger) Info(msg string, keysAndVals ...interface{}) {
 		Time:    timeNow(),
 		Message: msg,
 	}
+	// try and set the Caller in the entry.
```

Comment:
This commend doesn't express anything that is not already expressed by function call. 
Overall good name of function is always better then a explanation by comment. Comments get outdated much faster then function names.
```suggestion
```